ludwig/datasets/mixins/kaggle.py

```diff
@@ -28,7 +28,7 @@ def download_raw_dataset(self):
         kaggle.json file we lookup the passed in username and the api key and
         perform authentication.
         """
-        with self.update_env(KAGGLE_USERNAME=self.kaggle_username, KAGGLE_API_KEY=self.kaggle_api_key):
+        with self.update_env(KAGGLE_USERNAME=self.kaggle_username, KAGGLE_KEY=self.kaggle_key):
```

Comment:
I would revert this, let's keep on using the API key and deal with tests differently

Reply:
I have adapted the test that earlier failed and should probably work now. Since kaggle no longer recommends using `KAGGLE_API_KEY`, making this change to `KAGGLE_KEY` significant (c.f: https://github.com/Kaggle/kaggle-api#api-credentials)